executor.py

```diff
@@ -76,8 +76,10 @@ def search(
             else self._match_args
         )
 
+        traversal_right = parameters.get('traversal_right', '@r')
```

Comment:
can we have default ones at `__init__` time?

Reply:
yes. re-check pls